### PR TITLE
Update instructions on acceptance

### PR DESCRIPTION
**Pull request by davidlmobley**
Add info to editors on what happens when an article is accepted.

@dwsideriusNIST - perhaps this also needs something on HOW they should add the boilerplate/footers we decided on. You could add by a commit to this branch if you like.

### CODE REVIEW

File: _editorial_policies/02_editor_instructions.md [hunk 1]
```diff
@@ -48,3 +48,10 @@ Once an editor has handled the pre-review steps described above, the review proc
...
 - Makes a recommendation regarding acceptance, acceptance with minor or major revision, or rejection (keeping in mind the content of the presubmission letter), and communicates this recommendation to the Lead Editor who makes a final decision. Normally the Lead Editor will follow the handling editor's recommendations, but this process is helpful since the Lead will have approved the presubmission letter and thus needs to be involved if a less than favorable decision is to be made.
 - If the decision is that the authors must revise, provides appropriate direction (with the Lead Editor) about which comments to address if needed
+
+Following acceptance, the usual procedure is:
+- Have the author do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. You may assist with this as well if you like.
+- Have the author add LiveCoMS footer information and provide a PDF for posting
+- Post the PDF as an ASAP article
+- When assigned to an issue, get the DOI and article number, have the author place these into a PDF, and update the PDF
```
Comment (dmzuckerman) [inline _editorial_policies/02_editor_instructions.md]: Is it obvious in Scholastica how to replace a pdf?  If not, is there a link we can point to?

Comment (mrshirts) [reply _editorial_policies/02_editor_instructions.md]: Probably the managing editor (or whoever has journal access) -- that might be doable by any editor.  But as long as the editor can let us know it's ready, it's trivial to swap out the pdf.

File: _editorial_policies/02_editor_instructions.md [hunk 2]
```diff
@@ -48,3 +48,10 @@ Once an editor has handled the pre-review steps described above, the review proc
...
 - Makes a recommendation regarding acceptance, acceptance with minor or major revision, or rejection (keeping in mind the content of the presubmission letter), and communicates this recommendation to the Lead Editor who makes a final decision. Normally the Lead Editor will follow the handling editor's recommendations, but this process is helpful since the Lead will have approved the presubmission letter and thus needs to be involved if a less than favorable decision is to be made.
 - If the decision is that the authors must revise, provides appropriate direction (with the Lead Editor) about which comments to address if needed
+
+Following acceptance, the usual procedure is:
+- Have the author do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. You may assist with this as well if you like.
+- Have the author add LiveCoMS footer information and provide a PDF for posting
```
Comment (mrshirts) [inline _editorial_policies/02_editor_instructions.md]: Is it clear what "LiveCoMS footer information" is?  I'm not even sure.

File: _editorial_policies/02_editor_instructions.md (hunk 1 above)
Comment (mrshirts) [reply _editorial_policies/02_editor_instructions.md]: I think some of these concerns can be fixed if it's clarified who is supposed to do which thing (editor, author, managing editor, etc.)

File: _editorial_policies/02_editor_instructions.md (hunk 2 above)
Comment (davidlmobley) [reply _editorial_policies/02_editor_instructions.md]: It's the boilerplate stuff Dan added options for to the template. What do you want to call it?

Comment (dwsideriusNIST) [reply _editorial_policies/02_editor_instructions.md]: My suggested changes above (in a comment, not a revised PR) drop the term "footer information" and just refer to the footer (which I think everyone will understand) and then specifically what the footer contains. I was on a different PC, couldn't make a revised PR.

**Comment by davidlmobley** [comment]
Done, @mrshirts and @dwsideriusNIST . Can one of you do a final check and merge?

**Comment by davidlmobley** [comment]
@dwsideriusNIST we should put you on that list. :) 

In terms of doing revisions to PRs -- the way it works is that you 'git fetch' the repo, check out the branch, and then just commit directly to it. Alternatively, you git fetch the repo, check out the branch, make a new branch based on it, make changes in that branch, and then do a PR from your new branch TO THE ORIGINAL BRANCH (rather than to master).

**Pull request merged**